oshi-core/src/main/java/oshi/software/os/FileSystem.java

```diff
@@ -61,4 +61,12 @@ public interface FileSystem {
      * @return The maximum number of file descriptors if available, 0 otherwise.
      */
     long getMaxFileDescriptors();
+
+    /**
+     * The maximum number of open file descriptors per process. This returns the system-wide limit. The actual limit of
+     * the calling process may be lower.
+     *
+     * @return The maximum number of file descriptors of every process if available, 0 otherwise.
```

Comment:
Similar to the above comment, I'd say "each process, unless configured to a lower value" or similar.